css/common.css

```diff
@@ -1,75 +1,80 @@
-.footer{
-	width: 100%;
-	
-	line-height: 100px;
-	text-align: center;
-	color: #4d4d4d;
-	background-color: #bababa;	
+.footer {
+    width: 100%;
+    line-height: 100px;
+    text-align: center;
 }
 
-.logo{
-	width: 200px;
-	height: 37.40px;
-	margin-top: 20px;
-	margin-left: 50px;
-	margin-right: 150px;
-
-	float: left;
-	background-image: url('https://www.softvision.ro/images/common/logo-v2.png');
-	background-size: contain;
+.footerSkin {
+    color: #4d4d4d;
+    background-color: #bababa;
 }
 
-.header{
-	width: 100%;
-	height: 80px;
+.logo {
+    width: 200px;
+    height: 37px;
+    margin-top: 20px;
+    margin-left: 50px;
+    margin-right: 150px;
+    float: left;
+}
 
-	box-shadow: 0px 3px 10px grey;
+.logoSkin {
+    background-image: url('https://www.softvision.ro/images/common/logo-v2.png');
+    background-size: contain;
 }
 
-.headerSpan{
-	margin-top: 30px;
-	float: right;
+.header {
+    width: 100%;
+    height: 80px;
 }
 
-.formatted{
-	color: black;
-	font-size: 22px;
+.headerSkin {
+    box-shadow: 0 3px 10px grey;
 }
 
-.blue {
-	color: #1ab2ff;
+.headerSpan {
```

Comment:
do not put the tag name in the name of the class. The class can be used on any element